src/arch/riscv/isa/formats/standard.isa

```diff
@@ -339,7 +339,7 @@ def template CSRExecute {{
         }
 
         RegIndex midx = csr_data_it->second.physIndex;
-        const std::string& csrName = csr_data_it->second.name;
+        std::string csrName = csr_data_it->second.name;
```

Comment:
(nit) Is any reason to change it to copy the string from csr_data to csrName?

Reply:
It can no longer be const or reference due to possible remapping (reassignment) by swapToVirtCSR method.